src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/Hashing/BitOperations.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Copied from https://github.com/dotnet/runtime/blob/f1d463f46268382a8287d71aea929dadaa5dfef5/src/libraries/System.IO.Hashing/src/System/IO/Hashing/BitOperations.cs#L1C1-L18C2
+// Remove once we can actually add a reference to System.IO.Hashing v8.0.0
```

Comment:
Copied these files effectively unchanged.  All i did was:

1. add the 'copied from' line
2. add `<auto-generated>` in files that were failing linting.  

I also made 1-2 *tiny* tweaks where needed.  will call those out.

Reply:
Can we file and issue to delete this and then reference the issue in all of the files that need to be deleted? I'll assign that issue to myself and take care of deleting all this once we move to the 8.0.0 version of the core plat libs.

Reply:
@jaredpar will do